recipes/material-design-lite/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "material-design-lite" %}
+{% set version = "1.3.0" %}
+{% set sha256 = "4b4ab9845dba0e6a6fda172d69aff6da72c3e429dc1f3d30fa0935d315f40cb6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/google/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: generic
+  script: npm install -g .
+
+requirements:
+  build:
+    - nodejs
+  run:
```

Comment:
Can you remove this line?

Reply:
Thanks for the review. I have removed that line.